modules/reference/pages/kube-apis/conforma.adoc

```diff
@@ -0,0 +1,279 @@
+// Generated documentation. Please do not edit.
```

Comment:
This is new doc, it shoule be added into nav page https://github.com/konflux-ci/docs/blob/main/modules/reference/nav.adoc

However I see there enterprise contract page, should be enterprise contract page removed/replaced by conforma?

Reply:
Good point, I would change the nav.doc so it points to the conforma honestly. Let me change it.

Reply:
Agreed, removed.